### PR TITLE
[DOC] Fix interpreter version in doc

### PR DESCRIPTION
**Pull request by zjffdu**
### What is this PR for?
Change the version from 0.7.0 to 0.8.0

### What type of PR is it?
[Documentation]

### Todos
* [ ] - Task

### What is the Jira issue?
* No jira created

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by zjffdu** [comment]
@felixcheung  Could you help review it ? Thanks

File: docs/usage/interpreter/installation.md [hunk 1]
```diff
@@ -58,48 +58,42 @@ Zeppelin support both Scala 2.10 and 2.11 for several interpreters as below:
...
+#### Spark interpreter for scala 2.10 and 2.11 
+Starting from Zeppelin 0.8.0, Zeppelin support both scala 2.10 and 2.11 in one zeppelin instance. You don't need to install different scala specific interpreter for spark. 
+It would pick up the right scala specific interpreter for the spark distribution where `SPARK_HOME` point to.
+
+Here's the command to install spark interpreter
 
 ```
-rm -rf ./interpreter/spark
-./bin/install-interpreter.sh --name spark --artifact org.apache.zeppelin:zeppelin-spark_2.10:0.7.0
+./bin/install-interpreter.sh --name flink --artifact org.apache.zeppelin:spark-interpreter:0.8.0
```
Comment (felixcheung) [inline docs/usage/interpreter/installation.md]: `--name flink`?

File: docs/usage/interpreter/installation.md [hunk 2]
```diff
@@ -58,48 +58,42 @@ Zeppelin support both Scala 2.10 and 2.11 for several interpreters as below:
...
 ```
 
-<br />
-Once you have installed interpreters, you need to restart Zeppelin. And then [create interpreter setting](./overview.html#what-is-zeppelin-interpreter) and [bind it with your notebook](./overview.html#what-is-zeppelin-interpreter-setting).
```
Comment (felixcheung) [inline docs/usage/interpreter/installation.md]: no need to restart?

Comment (zjffdu) [reply docs/usage/interpreter/installation.md]: Fixed

**Comment by zjffdu** [comment]
Thanks for review @felixcheung  will merge it soon

**Pull request closed without merging**